app/models/location.rb

```diff
@@ -15,6 +15,69 @@ def self.primary(city_name = ENV['PRIMARY_CITY_NAME'])
   def weather
     response = ::Clients::WeatherClient.forecast(self).dup
     response['moonPhase'] = moon_phase
+    
+   response['current']['weather'][0]['hourly'] = response['hourly']
+
+   weather_info = {"rain" => "response['hourly'][2]['pop']" , "snow" => "response['hourly'][2]['snow']"}
```

Comment:
I don't think we want the quotes around the values. We want `"rain"` to be the value in the response, not the literal string `"response['hourly'][2]['pop']"`.